addons/rose/app/styles/rose/components/header/_dropdown.scss

```diff
@@ -10,28 +10,9 @@
 $chevron: "data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%23bac1cc' xmlns='http://www.w3.org/2000/svg'><path fill-rule='evenodd' clip-rule='evenodd' d='M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6-1.41-1.41z'></path></svg>";
 
 .rose-header {
-  .rose-dropdown {
-    &,
-    &[open] {
-      .rose-dropdown-trigger {
-        color: #{sass.color(ui-gray, 300)};
-        background: url($chevron) no-repeat right sizing.rems(s) center;
-        background-size: sizing.rems(m);
-
-        $y-padding: sizing.rems(xxxs);
-        $right-padding: sizing.rems(xs) + sizing.rems(l);
-
-        padding: $y-padding $right-padding $y-padding sizing.rems(s);
-
-        &.has-icon-only {
-          padding-left: sizing.rems(xs);
-        }
-
-        &:hover,
-        &:focus {
-          background-color: transparent;
-        }
-      }
-    }
+   .hds-dropdown > button {
```

Comment:
this is a temp fix, we will be retiring this header component when we integrate [hds side nav](https://helios.hashicorp.design/components/app-side-nav?tab=code) in desktop